tests/src/integration/tests/test_dse_auth.cpp

```diff
@@ -203,7 +202,6 @@ class AuthenticationTest : public DseIntegration {
     Cluster cluster = dse::Cluster::build()
                           .with_plaintext_authenticator(username, password)
                           .with_contact_points(contact_points_)
-                          .with_hostname_resolution(true)
```

Comment:
These two usages are the only places where with_hostname_resolution() was actually used.  Manually confirmed this test passes for all supported versions of DSE after this change.